### DISCUSSION TITLE
First Release

### DISCUSSION THREAD

**Discussion by EranOfek** [EranOfek/AstroPack#81] [Q&A]
# AstroPack / MAATv2 first release

AstroPack / MAATv2 first release for MATLAB file exchange.

<hr /><em>This discussion was created from the release <a href='https://github.com/EranOfek/AstroPack/releases/tag/0.1.0.0'>First Release</a>.</em>

**Comment by EastEriq** [comment]
For the second one you will figure out a real name?
